examples/MHD/input_otv.i3d

```diff
@@ -93,6 +93,16 @@ nstat = 1             ! Size arrays for statistic collection
 
 /End
 
+!====================
+&MHDParam
+!====================
+
+mhd_equation = 'induction'
+Stuart = 17.54386 
```

Comment:
provide definition for all the different Reynolds numbers

Reply:
Resolved